### ISSUE TITLE
Question: Binding PieChart's Values to an ObservableCollection<T>

### ISSUE THREAD

**Issue by mrousavy** [Live-Charts/Live-Charts#759]
The PieChart control does not contain a `Values` property to bind to, so the `Series` property has to be used.

Which means, I have to create a `SeriesCollection` and fill it with `PieSeries` objects. This works fine, until the contents change, because the changes from my original `ObservableCollection<T>` are obviously not going to be updates to the `SeriesCollection`.

Now I implemented my own `SeriesCollection` and `PieSeries` objects, so they are now listening for changes, but this whole thing seems very fragile and having two lists with the same content seems pointless.

Is there any way I could bind my `ObservableCollection<T>` directly to the PieChart's values, and tell it what properties it should plot and which are used for the legend?

#### I am using

<table>
<tr>
    <td>Live-Charts version</td>
    <td>0.9.7</td>
</tr>
<tr>
    <td>.Net Version</td>
    <td>4.6.1</td>
</tr>
<tr>
    <td>Windows</td>
    <td>10 x64</td>
</tr>
</table>

**Comment by kardall** [comment]
The SeriesCollection inherits NoisyCollection which does have an OnChanged event fire.

So yes, an example of what I have that works is this:

XAML
`<lvc:PieChart Margin="10 10 10 10" Grid.Column="0" Grid.Row="1" Grid.ColumnSpan="2" Series="{Binding PieChartSeriesCollection}" DataClickCommand="{Binding DataClickCommand}" />`

ViewModel
`if(PieChartSeriesCollection != null) PieChartSeriesCollection.Clear();
    foreach(KeyValuePair<string, int> pair in DictionaryOfData) {  // Dictionary<string, int>
        PieChartSeriesCollection .Add(
            new PieSeries { 
                Title = $"{pair.Value} ({pair.Key})", 
                Values = new ChartValues<int> { pair.Value }, 
                DataLabels = true 
            });
    }`

Will update in real time as you add/remove entries on your chart.

**Comment by mrousavy** [comment]
Yes, but this requires me to run two seperate lists, one being the original ObservableCollection **which gets updated from somewhere else**, and one being the SeriesCollection, for which I have to write dozens of lines of code which updates it's PieSeries values once the ObservableCollection's values change.

I want to directly bind to an ObservableCollection, so that there is no workaround needed for a second list and change handlers on the original list.

**Comment by kardall** [comment]
You could have a Converter for the ObservableCollection that converts them from the collection to a PieSeries on the fly. That might be a way to do it.

**Comment by beto-rodriguez** [comment]
v 1.0 support this,  should be ready in this month

**Comment by mrousavy** [comment]
Awesome, I'll wait for v1.0 then. Maybe I can help developing in the meantime.. I'll take a look
